proversity_reports_script/report_backend/completion_report.py

```diff
@@ -113,6 +119,8 @@ def create_csv_file(self, course, body_dict):
                     writer.writerow(row)
 
             self.upload_file_to_storage(course, path_file)
+            update_sheets_data(path_file, spreadsheet_id)
+
```

Comment:
remove this empty space

Reply:
Addressed, but i think that two blank lines is better for code reading, although pep8 says the opposite.